File: test/export/php/07-subprocesos.php
<?php
	$stdin = fopen('php://stdin','r');
	echo funciondoble(5),PHP_EOL;
	noretornanada(3,9);
	$c = 0;
	porreferencia($c);
	echo $c,PHP_EOL;
	$a = array();
	$b = array();
	recibevector($a);
	recibematriz($b);
	function funciondoble($b) {
		$a = 2*$b;
		return $a;
	}
	function noretornanada($a, $b) {
		echo $a+$b,PHP_EOL;
	}
	function porreferencia(&$b) {
		$b = 7;
	}
	function recibevector($v) {
		global $stdin;
		$v[1]=rtrim(fgets($stdin),PHP_EOL);
	}
	function recibematriz($m) {
		global $stdin;
		$m[1][1]=rtrim(fgets($stdin),PHP_EOL);
	}
?>

File: test/export/c/07-subprocesos.c
#include<stdio.h>
#define SIN_TIPO float
float funciondoble(float b);
void noretornanada(SIN_TIPO a, SIN_TIPO b);
void porreferencia(float *b);
void recibevector(SIN_TIPO v[]);
void recibematriz(SIN_TIPO m[][]);
int main() {
	SIN_TIPO a[10];
	SIN_TIPO b[3][4];
	float c;
	printf("%s\n",funciondoble(5));
	noretornanada(3,9);
	c = 0;
	porreferencia(&c);
	printf("%f\n",c);
	recibevector(a);
	recibematriz(b);
	return 0;
}
float funciondoble(float b) {
	float a;
	a = 2*b;
	return a;
}
void noretornanada(SIN_TIPO a, SIN_TIPO b) {
	printf("%f\n",a+b);
}
void porreferencia(float *b) {
	(*b) = 7;
}
void recibevector(SIN_TIPO v[]) {
	scanf("%f",v[0]);
}
void recibematriz(SIN_TIPO m[][]) {
	scanf("%f",m[0][0]);
}
